packages/web-frontend/src/containers/OverlayDiv/components/ProjectPage/ProjectCard.js

```diff
@@ -126,7 +126,7 @@ export const ProjectCard = ({
       <Title>{name}</Title>
       <FullWidthRow>{description}</FullWidthRow>
       <Footer>
-        <Countries>{name === 'Disaster Response' ? 'Global' : names.join(', ')}</Countries>
+        <Countries>{name === 'Disaster Response' ? 'Global' : names.sort().join(', ')}</Countries>
```

Comment:
The fix should be on the server instead, if we always want them sorted alphabetically. Then again, it's kind of a presentational concern, so this is still approved :+1: